docs/application/tizen-studio/setup/hardware-accelerated-execution-manager.md

```diff
@@ -9,50 +9,45 @@ Hardware requirements:
 - Intel processor with support for VT-x, EM64T, and Execute Disable Bit
 - At least 1 GB of available memory (2 GB or more is preferred)
 
-> **Important**  
+> **Important** 
```

Comment:
In the md file, to make a line break. you should keep two spaces after `**Important**`. Please add one deleted space back again.